frontend/src/app/app.tsx

```diff
@@ -41,6 +46,13 @@ const App: React.FC = () => {
         void refreshUser();
     }, [dispatch]);
 
+    useEffect(() => {
+        if (authDataStatus === DataStatus.REJECTED) {
+            void storage.drop(StorageKey.TOKEN);
+            navigate(AuthApiPath.SIGN_IN);
+        }
+    }, [authDataStatus, navigate]);
+
```

Comment:
I believe we should have handling for this in error middleware, maybe it's worth to just fix it there?